websites/A/Animekimi/presence.ts

```diff
@@ -41,54 +26,72 @@ presence.on("UpdateData", async () => {
+    ep =
+      document.querySelector(
+        "#single > div.content > div.sheader > div.data > h1"
+      )?.textContent ?? "ไม่ทราบชื่อตอน",
+    search =
+      document.querySelector(
+        ".content.rigth.csearch > header > h1" ||
+          ".no-result.animation-2 > h2 > span"
+      )?.textContent ?? "ไม่พบสิ่งที่คุณกำลังค้นหา",
+    path = document.location,
     presenceData: PresenceData = {
       largeImageKey: "site",
       startTimestamp: browsingStamp
     };
 
   // Presence
-  if (document.location.pathname === "/")
-    presenceData.details = "อนิเมะอัพเดตล่าสุด";
+  if (path.search.includes("?s")) {
+    const searchinfo = search.split("ผลการค้นหา:");
+    presenceData.details = "กำลังค้นหา";
+    presenceData.state = searchinfo.pop();
+    presenceData.smallImageKey = "search";
+  } else if (path.pathname === "/") presenceData.details = "อนิเมะอัพเดตล่าสุด";
   else if (path.pathname.includes("genre")) {
-    presenceData.details = "ประเภท ";
-    presenceData.state = title;
-  } else if (path.pathname.includes("catalog")) {
-    presenceData.details = "หมวดหมู่ ";
+    presenceData.details = "ประเภท";
     presenceData.state = title;
-  } else if (path.pathname.includes("category")) {
-    presenceData.details = "หมวดหมู่ ";
-    presenceData.state = title;
-  } else if (path.pathname.includes("title")) {
-    presenceData.details = "หมวดหมู่ ";
+  } else if (path.pathname.includes("catalog" || "category" || "title")) {
+    presenceData.details = "หมวดหมู่";
     presenceData.state = title;
   } else if (path.pathname.includes("tag")) {
-    presenceData.details = "หมวดหมู่ ";
-    presenceData.state = playvdo;
+    const titletag = title.split("รวมอนิเมะ");
+    presenceData.details = "รวมอนิเมะ";
+    presenceData.state = titletag.pop();
   } else if (path.pathname.includes("release")) {
-    presenceData.details = "ปี ";
+    presenceData.details = "ปี";
     presenceData.state = title;
-  } else if (path.search.includes("search")) {
-    presenceData.details = "ค้นหา ";
-    presenceData.state = playvdo;
   } else if (path.pathname.includes("movies")) {
-    presenceData.details = "เดอะมูฟวี่ ";
+    presenceData.details = "เดอะมูฟวี่";
     presenceData.state = titlemovies;
-    let moive;
+    let movie;
     const timestamps = presence.getTimestamps(
       Math.floor(video.current),
       Math.floor(video.duration)
     );
-    if (titlemovies.includes("เดอะมูฟวี่")) {
-      const moiveinfo = titlemovies.split("เดอะมูฟวี่");
-      moive = moiveinfo.pop();
-
-      if (moive.includes("ซับไทย"))
-        moive = moive.replace("เดอะมูฟวี่ ซับไทย", "").trim();
-      else if (moive.includes("พากย์ไทย"))
-        moive = moive.replace("เดอะมูฟวี่ พากย์ไทย", "").trim();
-
-      moive = `เดอะมูฟวี่ ${moive}`;
-      presenceData.details = moive;
+    if (titlemovies.includes("มูฟวี่" || "เดอะมูฟวี่")) {
+      const movieinfo = titlemovies.split("มูฟวี่" || "เดอะมูฟวี่");
```

Comment:
Same here

Reply:
I fixed the presence It won't work the way I want.

Reply:
No but like that isn't how `||` works

Reply:
What should I do